src/components/task-step/index.cjsx

```diff
@@ -33,7 +33,6 @@ TaskStepLoaded = React.createClass
   propTypes:
     id: React.PropTypes.string.isRequired
     onNextStep: React.PropTypes.func.isRequired
-    goToStep: React.PropTypes.func.isRequired
```

Comment:
I did some checking, seems like it was extra to begin with -- we don't end up using it inside the steps themselves, but we do need it on the Task